artemis/modules/data/bruter_additional_paths.txt

```diff
@@ -12,15 +12,8 @@ graph
 database.sql
 adminbackups
 core
-wp-admin/install.php
```

Comment:
which one added nuclei template duplicates wp-admin?

Reply:
http/misconfiguration/installer/wp-install.yaml which is used as it's critical-severity